nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,105 @@
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Free to start (Express edition)
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Deployment considerations
+
+### Introduction
+Any NServiceBus endpoint operates and manages different types of data, those are:
+ * Business data - data used for implementing business capabilities
+ * Persistence data - infrastructure level data including: saga data, timeout manager state and message driven pub/sub information
+ * Transport data - queues and messages
+
+SQL Server Transport manages transport data and puts no constraints on the type and configuration of storage technology used for persistence and business data. It can work with any of available persisters i.e. NHibernate and RavenDB, as well as any storage mechanisms used inside message handlers.
+
+Building any system based on NServiceBus requires answering questions on how and where business, persistence and transport data will be stored. The purpose of this section is to define what needs to be considered from SQL Server Transport's perspective.
+ 
+NOTE: No matter what deployment options are chosen, there is one general rule that should always apply: **All transport data (input, audit and error queues) should reside on a single SQL Server instance**. Multi-instance deployments are technically possible but have been deprecated as of SQL Server Transport version 3. Please see: ###Section### for migration guidance.  
+
+### Transactionality
+SQL Server Transport supports all [transaction modes](/nservicebus/transports/transactions.md) in particular `TransactionScope` which enables `exactly once` message processing with usage of distributed transactions. However when transport, persistence and business data are all stored in a single SQL Server catalog it is possible to achieve `exactly-once` message delivery without need for distributed transactions (###Link to Sample###).
+
+NOTE: `Exactly once` message processing without distributed transactions can be achieved with any transport (and SQL Server Transport in particular) using [Outbox](/nservicebus/outbox/). It requires business and persistence data to share the storage mechanism but does not put any requirements on transport data storage.
+
+### Security 
+Security considerations for SQL Server Transport should follow the principle of least privilege. For that each endpoint should use dedicated SQL Server account with SELECT and DELETE permissions on it's input queues as well as INSERT permissions on other endpoint's input queues, audit queue and error queue. 
```

Comment:
Account is probably not the right term when it comes to SQL Server security. We should probably use the term principal, which can be a user, a login, or a role.